### PR TITLE
Irregular line break in kubelet/server/server.go

### PR DESCRIPTION
**Pull request by h4ghhh**
#### What this PR does / why we need it:
Looks very uncomfortable.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kubelet/server/server.go [hunk 1]
```diff
@@ -338,8 +338,7 @@ func (s *Server) InstallDefaultHandlers() {
 
 	s.addMetricsBucketMatcher("pods")
 	ws := new(restful.WebService)
-	ws.
-		Path("/pods").
+	ws.Path("/pods").
```
Comment (Pingan2017) [inline pkg/kubelet/server/server.go:341]: /ok-to-test

**Comment by fromanirh** [comment]
Thank you for your PR. It seems to only contain trivial edits. Please read our [trivial edits policy](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits). We encourage you to take a look at confirmed issues and bugs or issues marked `help-wanted`.

**Pull request closed without merging**